website/src/components/TutorialComponents/CodeBlock.tsx

```diff
@@ -36,13 +38,24 @@ export function GithubDiscussionBtn({ repo }) {
 // source code of LiveEditor that allows for code editing
 // a good starting point for customizing our own code editor
 
-const CodeEditor = (props) => {
+function CodeEditor(props: {
+    className: string,
+    code: string,
+    disabled: boolean,
+    language: string,
+    onChange: (code: string) => void,
+    prism: typeof Prism,
+    style?: CSSProperties,
+    theme: PrismTheme,
+    githubRepo: string | undefined,
```

Comment:
Or use ? For optimal fields 


    githubRepo?: string

Reply:
The field gets passed in all the time from the remark plugin, it's just that the value can be `undefined`.